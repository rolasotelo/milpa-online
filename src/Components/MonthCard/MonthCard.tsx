import React from "react";

interface Props {
  text: string;
  isCurrentMonth: boolean;
}

const MonthCard = (props: Props) => {
  return (
    <div
      className={`${
        props.isCurrentMonth ? "bg-mexicanGreen-light" : "bg-mexicanBoneLight"
      } w-48 h-8`}
    >
      {props.text.toUpperCase()}
    </div>
  );
};

export default MonthCard;
