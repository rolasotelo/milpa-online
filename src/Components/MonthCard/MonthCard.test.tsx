import React from "react";
import renderer from "react-test-renderer";

import MonthCard from "./MonthCard";

describe("<MonthCard />", () => {
  const defaultProps = { text: "Enero", isCurrentMonth: true };
  const wrapper = renderer.create(<MonthCard {...defaultProps} />);

  test("render", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
