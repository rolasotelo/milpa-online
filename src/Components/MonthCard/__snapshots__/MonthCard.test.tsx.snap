// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MonthCard /> render 1`] = `
<div
  className="bg-mexicanGreen-light w-48 h-8"
>
  ENERO
</div>
`;
